exercises/atbash-cipher/.meta/generator/atbash_cipher_case.rb

```diff
@@ -0,0 +1,27 @@
+require 'generator/exercise_case'
+
+class AtbashCipherCase < Generator::ExerciseCase
+  def workload
+    case property
+    when 'encode' then encode_workload
+    when 'decode' then decode_workload
+    else raise 'unexpected property encountered'
+    end
+  end
+
+  def encode_workload
+    [
+      "plaintext = '#{phrase}'",
+      "ciphertext = '#{expected}'",
+      "assert_equal ciphertext, Atbash.encode(plaintext)"
+    ].join("\n")
```

Comment:
I think `workload` can return an array of lines, so you don't need to join these yourself.

Reply:
Really? Cool. I'll get to it.

Reply:
Got it.